scripts/integration_test_contracts.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+# Wait until the height of the block is greater than or equal to 1.
+h="0"
+until [[ ${h} == '"1"' ]]
+do
+  h=$(fnsad query block 1 --node http://localhost:26658 --chain-id simd-testing --keyring-backend test | jq .block.header.height)
+done
+
+# send transaction to node
+cur_path=`pwd`
+
+artifacts_path="artifacts"
+
+## parameters
+CONTRACT_ADDRESS="link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8"
+FROM_ACCOUNT="link146asaycmtydq45kxc8evntqfgepagygelel00h"
+URL="http://localhost:26658"
+CODE_ID=1
+
+# This is a function that checks if the result is as expected.
+check_result() {
+    if [[ "$result" == *"failed"* ]]; then
+      exit 1
```

Comment:
how about echoing why it fails to STDERR before exit?

Reply:
fixed here.
[fix: change check_result to output error message](https://github.com/Finschia/finschia-wasm/pull/3/commits/76b9fd37c44a243e8568656a92def16aa62b33e7)